<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newManualOperation</class>
 <widget class="QDialog" name="newManualOperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="X_Jog_L">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>73</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X -</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>手动速度：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Y_Jog_R">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>122</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Y +</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StopAllAxis">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="X_Home">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>73</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>回零</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Z_Jog_L">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Z -</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_StepDistance">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Y_Jog_L">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>122</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Y -</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_ManualSpeed">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckBox_Step">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>步进：</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>机器状态：</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z轴位置：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="X_Position">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y轴位置：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Z_Position">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X轴位置：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Y_Position">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>输入口状态：</string>
    </property>
    <widget class="QCheckBox" name="CheckBox_Org3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ORG 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckBox_Org1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ORG 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckBox_In4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IN 4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckBox_Org2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ORG 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckBox_In3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IN 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckBox_In1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IN 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckBox_In2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IN 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>输出口状态：</string>
    </property>
    <widget class="QCheckBox" name="CheckBox_Out1_Status">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>OUT 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckBox_Out2_Status">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>OUT 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckBox_Out3_Status">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>OUT 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckBox_Out4_Status">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>OUT 4</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Z_Jog_R">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>170</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Z +</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Z_Home">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>170</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>回零</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;mm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="X_Jog_R">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>73</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X +</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Y_Home">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>122</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>回零</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>输出口选择：</string>
   </property>
   <widget class="QCheckBox" name="CheckBox_Out1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OUT 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBox_Out2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OUT 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBox_Out3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OUT 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBox_Out4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OUT 4</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;mm/s&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_OpenAutoControl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Century&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_RunOnTime">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>上电自动运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_IOStatus">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>165</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>IO扩展</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_DeleteProgram_Controller">
   <property name="geometry">
    <rect>
     <x>619</x>
     <y>99</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除程序</string>
   </property>
  </widget>
  <widget class="newcombobox" name="SelectProgram_Controller">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>58</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>40</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>控制卡内程序：</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>newcombobox</class>
   <extends>QComboBox</extends>
   <header>newcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
